fpmsyncd/routesync.cpp

```diff
@@ -886,6 +966,62 @@ bool RouteSync::getIfName(int if_index, char *if_name, size_t name_len)
     return true;
 }
 
+/*
+ * Get next hop List
+ * @arg route_obj     route object
+ *
+ * Return concatenation of NHs: nh0 + "," + nh1 + .... + "," + nhN
+ */
+string RouteSync::getNextHopList(struct rtnl_route *route_obj)
```

Comment:
Please add tests for this API. For example, provide different values of "route_obj" and compare with the expected string.

Reply:
Please refer to the tests provided in sonic-swss/tests/test_route.py:TestMplsRoute

Reply:
Tests are now moved to new location: sonic-swss/tests/test_route.py:TestMplsRouteFpm